cypress/integration/frontpageAmpSpec.js

```diff
@@ -0,0 +1,95 @@
+import worldServices from '../support/worldServices';
+import { getElement } from '../support/bodyTestHelper';
+import { testResponseCode, checkCanonicalURL } from '../support/metaTestHelper';
+import { describeForLocalOnly } from '../support/limitEnvRuns';
+
+// TODO Enable all disabled tests below once bbc/simorgh#1906 has been merged.
+//   The Metadata container performs some AMP work, so the AMP on the front page
+//   will not be valid until the Metadata container has been added into the front
+//   page container. 🙃
+describeForLocalOnly('AMP Tests on a .amp page', () => {
+  // eslint-disable-next-line no-undef
+  before(() => {
+    cy.visit(`${worldServices.igbo.url}.amp`);
+  });
+
+  describe('AMP Status', () => {
+    it('should return a 200 response', () => {
+      testResponseCode(`${worldServices.igbo.url}.amp`, 200);
+    });
+  });
+
+  it('should error gracefully', () => {
+    testResponseCode(`${worldServices.igbo.url}.cake`, 404);
```

Comment:
Can this file not loop on all 3 services? all the same logic, but rather than `worldServices.igbo.url`, loop on it and then to `service.url`

Reply:
I was under the impression that the current strategy was to avoid this because it would lengthen the e2e tests, especially when all 40ish WS sites are added. Has this changed?

Reply:
Ahhh thats true, fairs :)